releasenotes/notes/optimize-1q-gates-decomposition-ce111961b6782ee0.yaml

```
@@ -0,0 +1,13 @@
+---
+features_transpiler:
+  - |
+    Added a new method :meth:`.DAGCircuit.control_flow_ops` which provides a fast
+    path to get all the :class:`.DAGOpNode` in a :class:`.DAGCircuit` that
+    contain a :class:`.ControlFlowOp`. This was possible before using the
+    :meth:`.DAGCircuit.op_nodes` method and passing the ``ControlFlowOp`` class
+    as a filter, but this new function will perform the same operation but
+    perform it faster.
```

small suggestion to avoid the double "but":

```suggestion
    :meth:`.DAGCircuit.op_nodes` method and passing the :class:`.ControlFlowOp` class
    as a filter, but this new function will perform the operation faster.
```